CMSIS/Core_A/Source/irq_ctrl_gic.c

```diff
@@ -148,6 +149,10 @@ __WEAK int32_t IRQ_SetMode (IRQn_ID_t irqn, uint32_t mode) {
       status = -1;
     }
 
+    valnn = (mode & IRQ_MODE_MODEL_Msk);
+    if (valnn == IRQ_MODE_MODEL_1N)
+    	cfg |= 1;   // 1-N model
```

Comment:
Please add curly braces around the block.

Reply:
done.